logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java

```diff
@@ -87,11 +95,11 @@ public void error(Supplier<String> msgSupplier) {
 
   @Override
   public void error(String format, Throwable t) {
-    logger.error(() -> getLogMessagePrefix() + format, t);
+    logger.error(() -> getLogMessagePrefix() + format + System.lineSeparator() + throwableToString(t));
```

Comment:
Isn't "\n" equivalent to `System.lineSeparator()`? Although not sure how about Windows where EOL is represented with 2 chars :thinking:

Reply:
Would be nice to check, I'm pretty curious actually how Java handles that

Reply:
IIRC `System.lineSeparator()` is universal line separator that will be expanded to proper line separator regarding OS

Reply:
Ok, right https://stackoverflow.com/a/33505921/2750812; forbid `\n` in checkstyle then maybe

Reply:
Looks that `\n` resolves to a single line feed/`0x0a` byte on each OS, incl. Windows